WebXR Input Source - UI (#640)
playcanvas/developer.playcanvas.com: docs/user-manual/xr/input-sources.md

}
```

## UI

> The UI elements such as 3D screens, buttons, scroll areas, and other components work well with input sources, and events such as "click" will trigger regardless of input type: mouse, touch, or XR input source.

Reviewer: ```suggestion
UI elements such as 3D screens, buttons, scroll views, and other components work well with input sources. Events such as `click` will trigger regardless of input type: mouse, touch, or XR input source.
```

> By default, all input source rays will be used to check for interaction with UI components, but you can disable it using a flag:

Reviewer: ```suggestion
By default, all input source rays will be used to check for interaction with UI components, but you can disable this using a flag:
```

```javascript
inputSource.elementInput = false;
```

> You can also access a UI entity with which the input source has interacted:

Reviewer: ```suggestion
You can also access a UI entity with which an input source has interacted:
```

```javascript
const entity = inputSource.elementEntity;
if (entity) {

>     // a specific entity that input source has interacted with

Reviewer: ```suggestion
    // a specific entity that the input source has interacted with
```

}
```

> It is also possible to subscribe to ButtonComponent `select` events, that are fired only by XR input sources, similar to specific events of mouse or touch:

Reviewer: ```suggestion
It is also possible to subscribe to ButtonComponent `select` events, that are fired only by XR input sources, similar to specific mouse or touch events:
```

```javascript
entity.button.on('selectstart', (evt) => {

>     // this buttons is selected by evt.inputSource

Reviewer: ```suggestion
    // this button is selected by evt.inputSource
```
